lightning/src/ln/channel.rs

```diff
@@ -483,8 +483,8 @@ macro_rules! secp_check {
 
 impl<Signer: Sign> Channel<Signer> {
 	// Convert constants + channel value to limits:
-	fn get_holder_max_htlc_value_in_flight_msat(channel_value_satoshis: u64) -> u64 {
-		channel_value_satoshis * 1000 / 10 //TODO
+	fn get_holder_max_htlc_value_in_flight_msat(&self) -> u64 {
+		self.channel_value_satoshis * 1000 * self.holder_max_htlc_value_percentage / 100
```

Comment:
Should we bound by min(max(setting, 1), 90) (and update documentation to match)?

Reply:
Make sense.